data/latest/run-go-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
+# $1 - edgex-snap-testing test suite
+
+if [ -f edgex-snap-testing ]; then
+    rm -rf edgex-snap-testing
+fi
+
+git clone --depth 1 --branch v3 https://github.com/canonical/edgex-snap-testing.git
```

Comment:
Error running latest test suite in snap:
```
Cloning into 'edgex-snap-testing'...
warning: templates not found in /usr/share/git-core/templates
fatal: unable to find remote helper for 'https'
```

Reply:
Thanks. The templates cant be created:
```
mkdir: cannot create directory ‘/usr/share/git-core’: Read-only file system
```
Also the fatal error is about `git clone`.

Reply:
layouts works. New dependencies issues presents:
```
Cloning into 'edgex-snap-testing'...
...
# runtime/cgo
_cgo_export.c:3:10: fatal error: stdlib.h: No such file or directory
    3 | #include <stdlib.h>

```